src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -329,7 +329,7 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
   Loft: {
     description: 'Create a 3D body by blending between two or more sketches',
     icon: 'loft',
-    needsReview: true,
+    needsReview: false,
```

Comment:
This isn't exactly a fix. The way it was, somehow during the review step the confirm button wouldn't auto select but the selection argument would. Couldn't figure out why.

However, for loft we don't actually need to review anything, since it's only based on the selections. So here, in one click after selection we create the loft. All in all better but we're not fixing the root issue. Note that the issue doesn't happen on other multi-select flows like Shell that have another argument after selection.

Reply:
https://github.com/user-attachments/assets/a9d7cc47-89ff-49c9-a58c-2bbcb63c3b88

Reply:
That is super interesting!